VocaDbWeb/Scripts/Components/Event/EventDetails.tsx

```diff
@@ -635,10 +634,11 @@ const EventDetails = (): React.ReactElement => {
 			)
 			.catch((error) => {
 				if (error.response) {
-					if (error.response.status === 404) navigate('/Error/NotFound');
+					if (error.response.status === 404)
+						window.location.href = '/Error/NotFound';
 				}
 			});
-	}, [id, navigate]);

```

Comment:
Do not pass `navigate` here. `useNavigate` triggers `useEffect`. See also https://github.com/remix-run/react-router/discussions/8465.